common/element_handle.go

```diff
@@ -1106,6 +1076,60 @@ func (h *ElementHandle) queryAll(selector string, eval evalFunc) ([]api.ElementH
 	return els, nil
 }
 
+// SetChecked checks or unchecks an element.
+func (h *ElementHandle) SetChecked(checked bool, opts goja.Value) {
```

Comment:
Will `SetChecked` also be exposed as a `Locator` method? [Playwright has it](https://playwright.dev/docs/api/class-locator#locator-set-checked), so did you plan to add it later? If so, this refactor would be better suited for that PR.